includes/net-standard-table.md

```diff
@@ -8,7 +8,7 @@
 | Xamarin.Mac                | 3.0    | 3.0    | 3.0   | 3.0   | 3.0   | 3.0                | 3.0                | 3.8                 | 5.16 |
 | Xamarin.Android            | 7.0    | 7.0    | 7.0   | 7.0   | 7.0   | 7.0                | 7.0                | 8.0                 | 10.0 |
 | Universal Windows Platform | 10.0   | 10.0   | 10.0  | 10.0  | 10.0  | 10.0.16299         | 10.0.16299         | 10.0.16299          | TBD |
-| Unity                      | 2018.1 | 2018.1 | 2018.1| 2018.1| 2018.1| 2018.1             |  2018.1            | 2018.1              | TBD |
+| Unity                      | 2018.1 | 2018.1 | 2018.1| 2018.1| 2018.1| 2018.1             |  2018.1            | 2018.1              | 2021.2 (not yet released) |
```

Comment:
I would use beta moniker since that's what Unity uses for the version.

```suggestion
| Unity                      | 2018.1 | 2018.1 | 2018.1| 2018.1| 2018.1| 2018.1             |  2018.1            | 2018.1              | 2021.2b |
```

Reply:
Hmm, `2021.2b` is not descriptive enough; it doesn't specify in which beta version was support for .NET Standard 2.1 added. How about `2021.2b6` or `2021.2 Beta 6` (my favorite of the three)?

Another option is to just write `2021.2` and either immediately merge it and hope people would figure out that the version is not yet released, or wait and merge it until the stable version of Unity 2021.2 gets released.

Reply:
Hi @teo-tsirpanis I just updated it to 2021.2.0b6 which is used in their release notes.